Makefile

```diff
@@ -83,6 +83,25 @@ kubeval: _site
 	! grep -C3 -P "invalid|\t\*" filtered.out
 	rm filtered.out
 
+helm-tests: vendor bin/helm helm-tests/fv.test versions-values.yaml
+ifndef RELEASE_STREAM
+	$(error RELEASE_STREAM is undefined - run using make helm-tests RELEASE_STREAM=vX.Y)
+endif
+	./helm-tests/fv.test -chart-path=./_includes/$(RELEASE_STREAM)/charts/calico
+
+PACKAGE_NAME?=github.com/projectcalico/calico
+helm-tests/fv.test: $(shell find ./helm-tests -type f -name '*.go' -print)
```

Comment:
Since we're _not_ shelling out to docker any more, we shouldn't need to build this into a binary and then run it. Check how we usually run ut's - should work with some simple `go test ./helm-tests`

Reply:
You'll need to volume mount the helm binary into the go-tests container too